### PR TITLE
Update JDG versions and add Infinispan 7.2 and 8.0 plugins

### PR DESCRIPTION
**Pull request by alanfx**
Update jdg64 version to 6.4.1
Update jdg65 to GA version
Use -Pinfinispan80 to build Infinispan 8.0 plugin

### CODE REVIEW

**Comment by mcimbora** [comment]
@alanfx Hmm not sure why you introduced separate profile for 8.x, could you elaborate?

**Comment by rvansa** [comment]
@mcimbora @alanfx That profile really shouldn't be there since 8.0 has already Beta release.

**Comment by alanfx** [comment]
@rvansa and @mcimbora, ok I'll update the PR. Thanks

**Comment by alanfx** [comment]
PR updated

**Comment by mcimbora** [comment]
@alanfx Thanks Alan. There's one more thing I noticed - schema links are missing in the new plugins, could you add them? (Or I can handle that when pushing the changes, just let me know what works for you, thanks)

**Comment by alanfx** [comment]
@mcimbora updated with links. (Let me know if I did it right!)

**Comment by mcimbora** [comment]
@alanfx Unfortunately I still don't see schema links in infinispan72 & infinispan80 plugins. Maybe a wrong rebase? Each plugin should have a symbolic link to target/distribution/RadarGun-2.1.0-SNAPSHOT/schema/pluginxyz.xsd

**Comment by alanfx** [comment]
@mcimbora Yeah, I see the same thing. Let me try again, because I created the link files

**Comment by alanfx** [comment]
@mcimbora hm, the xsd file was added, instead of the link file...

**Comment by alanfx** [comment]
@mcimbora ok, now the link files are there with relative paths

**Comment by rvansa** [comment]
Veni, vidi, integrated.
